chia/plot_sync/receiver.py

```diff
@@ -114,6 +120,8 @@ def reset(self) -> None:
         self._keys_missing.clear()
         self._duplicates.clear()
         self._total_plot_size = 0
+        self._total_effective_plot_size = float(0)
```

Comment:
Why do we need to add this to the receiver since its the same as `_total_plot_size` just times a constant factor which is only relevant for display purpose?